src/test/java/pl/coderslab/charity/service/DonationServiceTest.java

```diff
@@ -0,0 +1,98 @@
+package pl.coderslab.charity.service;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import pl.coderslab.charity.entity.Donation;
+import pl.coderslab.charity.repository.DonationRepository;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+public class DonationServiceTest {
+
+    @Mock
+    private DonationRepository donationRepository;
+
+    @InjectMocks
+    private DonationService donationService;
+
+    @Test
+    void getAllDonationsShouldReturnDonationList() {
+        Donation firstDonation = new Donation();
+        firstDonation.setId(1L);
+        Donation secondDonation = new Donation();
+        secondDonation.setId(2L);
```

Comment:
tworzenie obiektów dałbym do prytnej osobnej metodki, z List.of(firstDonation, secondDonation)